### ISSUE TITLE
Enhancement to Mac pList options

### ISSUE THREAD

**Issue by tsnelson1** [evshiron/nwjs-builder-phoenix#36]
nw-builder allowed more general plist options in its build process, and since switching to this project, I have a need to customize the LSUIPresentationMode key with a non-default integer value in my app, but cannot do so without manually doing it after the built is completed.

Is there any way you can add a generic default options object with just key: value combinations?

**Comment by evshiron** [comment]
Fixed in `nwjs-builder-phoenix` 1.12.0.

You should be able to do this by modifying `build.mac.plistStrings`. See [the sample project](https://github.com/evshiron/nwjs-builder-phoenix/tree/master/assets/project) and [Options](https://github.com/evshiron/nwjs-builder-phoenix/blob/master/docs/Options.md) for more information.

**Comment by zkrige** [comment]
Can you give an example of how `CFBundleDocumentTypes` would work? How would I use the config option in the plist to generate this in the target plist file

```
	<key>CFBundleDocumentTypes</key>
	<array>
        <dict>
            <key>CFBundleTypeName</key>
            <string>Document</string>
            <key>CFBundleTypeRole</key>
            <string>Viewer</string>
            <key>LSHandlerRank</key>
            <string>Owner</string>
            <key>LSItemContentTypes</key>
            <array>
                <string>com.adobe.pdf</string>
                <string>org.idpf.epub-container</string>
                <string>org.idpf.epub-folder</string>
                <string>com.apple.ibooks.epub</string>
            </array>
        </dict>
    </array>
```
